builders/tooling/oracles/acurast.md

```diff
@@ -0,0 +1,146 @@
+
+Note that to obtain a readable price from the price feed, it's essential to adjust for the feed's decimal places, which can be determined using the `decimals()` method. For instance, if the price feed returns a value of `51933620000`, you'll need to move the decimal point six places to accurately reflect the price. In this example, it corresponds to a Bitcoin price of `$51,933.62` at the time of writing. 
+
+### Supported Assets {: #supported-assets }
+
+By its design, Acurast can support the price feed of any arbitrary asset that is accessible by an API. The API request that powers the demo price feed is as follows: 
+
+```bash
+curl "https://api.binance.com/api/v3/ticker/price?symbol=BTCUSDT"
+```
+
+Upon running the above command in your terminal, you'll see a result that resembles the following:
+
+--8<-- 'code/builders/tooling/oracles/acurast/terminal/api.md'
+
+
+!!! note
+    This simple example of fetching a price feed relies on a single source of price feed data from one exchange. You can build a more complex job script that aggregates pricing data from multiple sources. 
+
+
+## Designing and Launching Your Own Price Feed {: #designing-and-launching-your-own-price-feed }
+
+You can build and launch your own Acurast price feed on your Tanssi EVM-compatible Appchain. Please be advised that the steps shown in this tutorial are unaudited, unverified, and for demonstration purposes only - it's highly recommended that you [contact the Acurast team directly](https://acurast.com/){target=\_blank} as they can assist you with launching price feeds on your Appchain to ensure the integrity of the deployment process.
+
+To launch an Acurast price feed, you need two key components: a smart contract and a script. In the prior example of [Interacting with the BTC/USD price feed](#interacting-with-price-feeds-demo-evm-appchain) on the Demo EVM Appchain, the generic Chainlink interface is used because it is a more straightforward example for demonstration purposes. The underlying smart contract that powers that price feed conforms to the [Chainlink Aggregator interface](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol){target=\_blank}, but the demo contract has additional components worthy of discussion. You can find both the demo contract and script at the [Github repo for the Acurast demo BTC/USD price feed](https://github.com/Acurast/acurast-evm-oracle-sample/tree/main){target=\_blank}.
+
+The demo contract, `InsecureDummyPriceFeed.sol`, emits an event when the price is updated and when a new round begins. The `setPrice` method is insecure, as shown in this demo smart contract, but it is provided to show you where you might add logic like aggregation consensus, access control checks, and other parameters.
+
+??? code "InsecureDummyPriceFeed.sol"
+
+    ```solidity
+    --8<-- 'code/builders/tooling/oracles/acurast/InsecureDummyPriceFeed.sol'
+    ```
+
+
```

Comment:
```suggestion
???+ code "InsecureDummyPriceFeed.sol"

    ```solidity
    --8<-- 'code/builders/tooling/oracles/acurast/InsecureDummyPriceFeed.sol'
    ```

```

Reply:
lets just expand to start with since they'll definitely need to interact with this if they're following along in this section